lib/msf/core/post/android/system.rb

```diff
@@ -0,0 +1,39 @@
+# -*- coding: binary -*-
+
+require 'msf/core/post/common'
+require 'msf/core/post/file'
+require 'msf/core/post/unix'
+
+module Msf
+class Post
+module Android
+module System
+
+  include Msf::Post::Common
+  include Msf::Post::File
+
+  public
+
+  # Returns system information from build.prop.
+  #
+  # @return [Hash] System information.
+  def get_sysinfo
```

Comment:
This looks a lot different from what meterpreter normally calls `sysinfo`. I'd strongly prefer this be called something else, perhaps `def build_prop`